forgevtt-module.js

```diff
@@ -1985,27 +2029,87 @@ class ForgeVTT_FilePicker extends FilePicker {
             }
         }
         // Sort the share-URL buckets by name
-        buckets.push(...sharedBuckets.sort((a, b) => a.label.localeCompare(b.label)));
+        sharedBuckets.sort((a, b) => a.label.localeCompare(b.label));
+        buckets.push(...sharedBuckets);
+        if (ForgeAPI.lastStatus) {
+            this._forgeBucketIndex = buckets;
+        }
```

Comment:
We only cache the buckets if the API status is ready, because the super constructor will first call this method before the user data is available.